assets/js/components/Config/ModbusProxyModal.vue

```diff
@@ -1,27 +1,305 @@
+								}))
+							"
+							transparent
+						/>
+					</FormRow>
+					<div class="align-items-center d-flex mb-4 justify-content-between">
+						<div>
+							<input
+								id="networkConnectionRTU"
+								v-model="connection.Settings.RTU"
+								class="form-check-input"
+								type="checkbox"
+							/>
+							<label class="form-check-label ms-2" for="networkConnectionRTU">
+								Use Modbus RTU over TCP
+							</label>
+						</div>
+						<button
+							type="button"
+							class="btn btn-sm btn-outline-secondary border-0"
+							aria-label="Remove"
+							tabindex="0"
+							@click="removeConnection(values, connection)"
+						>
+							<shopicon-regular-trash
+								size="s"
+								class="flex-shrink-0"
+							></shopicon-regular-trash>
+						</button>
+					</div>
+				</div>
+				<div class="d-flex align-items-center">
+					<button
+						type="button"
+						class="d-flex btn btn-sm btn-outline-secondary border-0 align-items-center gap-2 evcc-gray"
+						data-testid="networkconnection-add"
+						tabindex="0"
+						@click="addConnection(values, DEFAULT_NETWORK_CONNECTION)"
+					>
+						<shopicon-regular-plus
+							size="s"
+							class="flex-shrink-0"
+						></shopicon-regular-plus>
+						Add network connection
+					</button>
+				</div>
+			</div>
+			<div>
+				<h6>Serial Connection</h6>
+				<p>
+					The device is connected directly via an RS485 adapter (Modbus RTU). Check the
+					device configuration and read the relevant user manuals, data sheets, or system
+					settings for further details.
+				</p>
+				<div
+					v-for="(connection, index) in values.filter(
+						(item) => 'Device' in item.Settings
+					) as ModbusProxy<ModbusProxySerialConnection>[]"
+					:key="index"
+				>
+					<div class="d-none d-lg-block">
+						<hr class="mt-5" />
+						<h5>
+							<div class="inner mb-3">Serial Connection #{{ index + 1 }}</div>
+						</h5>
+					</div>
+
+					<FormRow id="serialConnectionDevice" label="Device" example="/dev/ttyUSB0">
+						<input
+							id="serialConnectionDevice"
+							v-model="connection.Settings.Device"
+							class="form-control"
+							required
+						/>
+					</FormRow>
+					<FormRow
+						id="serialConnectionPort"
+						label="Port"
+						example="5022"
+						help="The local TCP/IP port under which a connection is provided as a proxy server."
+					>
+						<input
+							id="serialConnectionPort"
+							v-model="connection.Port"
+							class="form-control"
+							required
+						/>
+					</FormRow>
+					<FormRow id="serialConnectionBaudrate" label="Baudrate" example="38400">
+						<input
+							id="serialConnectionBaudrate"
+							v-model="connection.Settings.Baudrate"
+							class="form-control"
+							required
+						/>
+					</FormRow>
+					<FormRow id="serialConnectionComset" label="Comset" example="8N1">
+						<input
+							id="serialConnectionComset"
+							v-model="connection.Settings.Comset"
+							class="form-control"
```

Comment:
can be a select. possible values are `8N1, 8E1, 8N2`. See https://github.com/Maschga/evcc/blob/master/assets/js/components/Config/DeviceModal/Modbus.vue#L221